python_lab3.py

```diff
@@ -0,0 +1,25 @@
+def decorator_for_html_tags(f):
+    def wrap(*args):
+        text = f(*args)
+        tags = {"paragraph": "p", "header": "h1", "hyperlink": "a", "article": "article", "body": "body", "button": "button", "column": "col", "directory list": "dir", "section": "div", "footer": "footer", "image": "img", "list item": "li", "metadata": "meta", "navigation links": "nav", "short quotation": "q", "script": "script"}
```

Comment:
you don't have to list all the html tags here, use decorator parameters to pass any tag